### ISSUE TITLE
Shouldn't need headers when writing array.

### ISSUE THREAD

**Issue by shaunc** [maxogden/csv-write-stream#19]
Currently, if there are no headers and `sendHeaders: false` is specified, writing an array results in ` Error: no headers specified`. However, in this situation can we not just write the array directly? 

If it is simplest to implement this way, perhaps, call `_compile` with a range of indexes corresponding to elements in the first row sent?

**Comment by thehappycoder** [comment]
Passed array can be an array of rows (arrays or objects) or an array of values (which is basically a row)

**Comment by CryptoKiddies** [comment]
yes, is there away to turn off the headers requirement? I don't want to specify the header for each value in a row if possible thanks.
